packages/core/hooks/useFileAccessContextMenu.ts

```diff
@@ -157,22 +157,26 @@ export default (filters?: FileFilter[], onDismiss?: () => void) => {
                     },
                     subMenuProps: {
                         items: [
+                            {
```

Comment:
The "move" option that contains these should only be rendered when the data source is "AICS FMS" (there is a constant for that value)

Reply:
resolved in [f52d31c](https://github.com/AllenInstitute/biofile-finder/pull/311/commits/f52d31c7091c7f8f43dfd4fd1af3aabe80ec94fe)